src/qz/common/TrayManager.java

```diff
@@ -452,11 +452,7 @@ public void setServer(final Server server, final AtomicBoolean running, final At
                 public void run() {
                     try {
                         setDangerIcon();
-                        running.set(false);
-                        securePortIndex.set(0);
-                        insecurePortIndex.set(0);
```

Comment:
These are passed in as parameters and only used in this one spot, so if they aren't being used here anymore, we should probably clean up the method's signature too.

Reply:
Addressed this. `insecurePortIndex` was still required due to `singleInstanceCheck`, but some refactoring might work there for it to not be necessary, instead using `getInsecurePortInUse()` and dropping `singleInstanceCheck` entirely.

But, that's outside the scope of this PR.